package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "dev": "webpack --mode development ./onboarding/src/index.js --output ./onboarding/static/js/main.js",
     "start": "webpack --mode development --watch ./onboarding/src/index.js --output ./onboarding/static/js/main.js",
-    "build": "webpack --mode production ./onboarding/src/index.js --output ./onboarding/static/js/main.js"
+    "build": "webpack --mode production ./onboarding/src/index.js --output ./onboarding/static/js/main.js",
+    "dev2": "webpack --mode development ./onboarding/src/employee.js --output ./onboarding/static/js/employee.js --config ./webpack.config.employee.js",
+    "start2": "webpack --mode development --watch ./onboarding/src/employee.js --output ./onboarding/static/js/employee.js --config ./webpack.config.employee.js",
```

Comment:
Zamiast "[...]2" proponuję zrobić coś bardziej intuicyjnego, np. "dev--employee", "start--employee".

Reply:
I totally agree,
but I couldn't find anything better quickly :/
If you have an interesting idea, feel free to share :-)